packages/@aws-cdk-testing/framework-integ/test/aws-opensearchservice/test/integ.opensearch.gp3.ts

```diff
@@ -25,7 +25,12 @@ class TestStack extends Stack {
   }
 }
 
-const app = new App();
+const app = new App({
+  context: {
+    // deploy will fail if this is not set to false unless zone awareness is enabled
+    '@aws-cdk/aws-opensearchservice:enableOpensearchMultiAzWithStandby': false,
```

Comment:
this is requred, otherwise the integ will fail as multi-az with standby enabled requires zoneawareness to be enabled.

Reply:
Is this related to the TLS change?

Reply:
Just as the comment said, If we don't disable this FF, which defaults true. The existing integ test won't deploy and will fail as zone awareness is disabled by default. Another option is to enable the zone awareness.

Reply:
> The existing integ test won't deploy and will fail as zone awareness is disabled by default.

Just checking - does this mean that the existing integ test (even without the TLS changes) is broken?

Reply:
reverted